src/routes/safe/components/Apps/confirmTransactions.js

```diff
@@ -38,9 +49,35 @@ const confirmTransactions = (
   closeModal: () => void,
   onConfirm: () => void,
 ) => {
+  const isTxValid = (t) => {
+    const isValidAddress = mustBeEthereumAddress(t.to) === undefined
```

Comment:
if `isValidAddress` is true, do we need these checks at all?
```
      t.to &&
      typeof t.to === 'string' &&
      isValidAddress 
```

if t.to is undefined or has a different type, it will be an invalid address, won't it?

Reply:
```
isValidAddress
isTxValid
```

how. do you define if valid goes in front of a noun or behind?

Reply:
done!